### PR TITLE
Team name and members added

### PR DESCRIPTION
**Pull request by starlit79**

### CODE REVIEW

**Review by hollisjamison — COMMENTED** [review]
You can find the mongo connection in src/database/mongo-common.js

Specifically these lines:

```
async function startDatabase() {
  const mongoDBURL = `mongodb://mlab2020:abc123def!@ds031617.mlab.com:31617/learningmongo`;
  const connection = await MongoClient.connect(mongoDBURL, {useNewUrlParser: true});
  database = connection.db();
}

File: HIGHLIGHTS.md [hunk 1]
```diff
@@ -7,3 +7,9 @@ What highlights can you come up with to share with the class?
 What do you really like? What can you find for fun facts related to any of these topics online?
 
 ## Cohort Highlights
+## 1. React storefront pulling in and showing data from another port such as week 10 API store manager.  Fetch() function
+Fun fact: The meaning of a response code from the server varies from API to API, and oftentimes checking response.ok might not be enough. For example, some APIs will return a 200 response even if your API key is invalid. Always read the API documentation!
```
Comment (SBucharelli) [inline HIGHLIGHTS.md:11]: This is interesting! I wasn't aware of this - thanks!

**Review by SBucharelli — APPROVED** [review]
You have great questions that I'd like to know more about as well.

File: HIGHLIGHTS.md [hunk 2]
```diff
@@ -7,3 +7,9 @@ What highlights can you come up with to share with the class?
...
+https://www.sitepoint.com/introduction-to-the-fetch-api/
+## 2. I like how easy it is to setup tables in react using bootstrap
```
Comment (xworld2000) [inline HIGHLIGHTS.md:13]: If this is so easy please show me!

**Pull request merged**